[2025-01-28T11:22:53.073+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-28T11:22:53.086+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T11:22:10.902666+00:00 [queued]>
[2025-01-28T11:22:53.092+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T11:22:10.902666+00:00 [queued]>
[2025-01-28T11:22:53.093+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2025-01-28T11:22:53.104+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): ingest_data> on 2025-01-28 11:22:10.902666+00:00
[2025-01-28T11:22:53.111+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_data_ingestion', 'ingest_data', 'manual__2025-01-28T11:22:10.902666+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/dag_data_ingestion.py', '--cfg-path', '/tmp/tmpjcdzvs_3']
[2025-01-28T11:22:53.113+0000] {standard_task_runner.py:105} INFO - Job 89: Subtask ingest_data
[2025-01-28T11:22:53.114+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=267) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-28T11:22:53.115+0000] {standard_task_runner.py:72} INFO - Started process 268 to run task
[2025-01-28T11:22:53.148+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T11:22:10.902666+00:00 [running]> on host 0711bf72f9ce
[2025-01-28T11:22:53.235+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_data_ingestion' AIRFLOW_CTX_TASK_ID='ingest_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-28T11:22:10.902666+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-28T11:22:10.902666+00:00'
[2025-01-28T11:22:53.236+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-28T11:22:53.248+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-28T11:22:53.249+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'ls /opt/***/data/']
[2025-01-28T11:22:53.259+0000] {subprocess.py:99} INFO - Output:
[2025-01-28T11:22:53.263+0000] {subprocess.py:106} INFO - yellow_2022_01_data.parquet
[2025-01-28T11:22:53.263+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-28T11:22:53.286+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-28T11:22:53.286+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_data_ingestion, task_id=ingest_data, run_id=manual__2025-01-28T11:22:10.902666+00:00, execution_date=20250128T112210, start_date=20250128T112253, end_date=20250128T112253
[2025-01-28T11:22:53.328+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-28T11:22:53.343+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-28T11:22:53.344+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
